standards/travis-ci.md

```diff
@@ -53,8 +53,9 @@ To configure Travis to deploy to Elastic Beanstalk:
 
  1. If you haven't already, run log in to Github through travis:
     `travis login --org`
- 2. Add encrypted credentials to `.travis.yml` by running this encrypted command (stored in nypl-digital-dev parameter store): https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Parameters:Name=%5BEquals%5Dproduction/travis/add_aws;sort=Name
- 3. Add `deploy` entries to `.travis.yml` for each deploy hook, e.g.:
+ 2. *From the root directory of your app,* run the encrypted command stored in nypl-digital-dev parameter store: https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Parameters:Name=%5BEquals%5Dproduction/travis/add_aws;sort=Name which It will automatically add encrypted credentials for AWS_ACCESS_KEY_ID_DEVELOPMENT and AWS_SECRET_ACCESS_KEY_DEVELOPMENT to `.travis.yml`.
```

Comment:
This will also encrypt AWS_SECRET_ACCESS_KEY_QA, AWS_SECRET_ACCESS_KEY_PRODUCTION, AWS_ACCESS_KEY_ID_QA, AWS_ACCESS_KEY_ID_PRODUCTION as well. Not only development.